src/main/java/org/tron/common/entity/NodeInfo.java

```diff
@@ -22,6 +24,8 @@
   /*machine information*/
   private MachineInfo machineInfo;
 
+  private Map<String, String> cheatWitnessInfoMap = new HashMap<>();
+
   public static class MachineInfo {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Reduce this class from 207 to the maximum allowed 25 or externalize it in a public class. ([squid:S2972](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2972))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1639&issues=AWalU-vKlHK86Hco_1fI&open=AWalU-vKlHK86Hco_1fI)

[ISSUE_KEY]: AWalU-vKlHK86Hco_1fI